event_model/__init__.py

```diff
@@ -571,8 +571,10 @@ def compose_event_page(*, descriptor, event_counter, data, timestamps, seq_num,
     return doc
 
 
-def compose_event(*, descriptor, event_counter, data, timestamps, seq_num,
+def compose_event(*, descriptor, event_counter, data, timestamps, seq_num=None,
                   filled=None, uid=None, time=None, validate=True):
+    if seq_num is None:
```

Comment:
Oh, thanks for this! I remember hitting this but apparently forgot to go back and fix it.